src/IPinfo/Utilities/CountryHelper.cs

```diff
@@ -30,10 +31,32 @@ internal static class CountryHelper
             return countries;
         });
 
+        /// <summary>
+        /// Lazy initialization for the EUcountry List from EUcountry json file.
+        /// </summary>
+        private static readonly Lazy<List<string>> eu_countries =
+        new Lazy<List<string>>(() =>
+        {
+            List<string> _eu_countries;
+            var assembly = Assembly.GetExecutingAssembly();
+            using (Stream stream = assembly.GetManifestResourceStream(EuCountriesJsonFilePathName))
```

Comment:
`EuCountriesJsonFilePathName` to be replaced with `EUCountriesJsonFilePathName`.